apps/crn-server/src/app.ts

```diff
@@ -417,7 +418,12 @@ export const appFactory = (libs: Libs = {}): Express => {
 
   /* istanbul ignore next */
   if (libs.sentryErrorHandler) {
-    app.use(libs.sentryErrorHandler());
+    const middleware = libs.sentryShouldHandleError
+      ? libs.sentryErrorHandler({
+          shouldHandleError: libs.sentryShouldHandleError,
+        })
+      : libs.sentryErrorHandler();
```

Comment:
```suggestion
      : libs.sentryErrorHandler({
         shouldHandleError
      });
```
?
otherwise the one from /utils/ would not get called in the real app